packages/gatsby/src/commands/build-utils.ts

```diff
@@ -59,14 +64,17 @@ const checkAndRemoveEmptyDir = (publicDir, pagePath) => {
   }
 }
 
-const sortedPageKeysByNestedLevel = pageKeys =>
+const sortedPageKeysByNestedLevel = (pageKeys: string[]): string[] =>
   pageKeys.sort((a, b) => {
     const currentPagePathValue = a.split(`/`).length
     const previousPagePathValue = b.split(`/`).length
     return previousPagePathValue - currentPagePathValue
   })
 
-const removePageFiles = ({ publicDir }, pageKeys) => {
+export const removePageFiles = async (
+  publicDir: string,
```

Comment:
This seems like it's changing signature from first arg being object to being just string. Please adjust usage in `build.js`

Reply:
@pieh
Thanks. I fixed it.